lib/tmuxinator.rb

```diff
@@ -13,9 +13,9 @@
 require "tmuxinator/project"
 require "tmuxinator/window"
 require "tmuxinator/version"
+require "tmuxinator/tmux_validator"
 
-module Tmuxinator
-end
+module Tmuxinator; end
```

Comment:
While this isn't really relevant to this PR, per se, it turns out that this line is actually unnecessary.